geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -28,6 +28,7 @@
 )
 
 
+LOG = logging.getLogger(__name__)
```

Comment:
I expect this will now fail flake8 tests.  I'll see about ignoring this line?

Reply:
What if we rework this a little bit? Something like this:

```python
optional_packages = {}
try:
    import numexpr as ne
    optional_packages['numexpr'] = True
except ImportError:
    optional_packages['numexpr'] = False

LOG = logging.getLogger(__name__)

for pkg, have in optional_packages.items():
    if not have:
        LOG.info(f"Failed to import {pkg} in {__file__}. If you need it, install it.")
```

Reply:
That solution doesn't work, but I have one that will. I'll put it together and see what you think.